Sources/SwiftSyntax/gyb_generated/SyntaxEnum.swift

```diff
@@ -486,7 +486,7 @@ public enum SyntaxEnum {
     case .parameterClause:
       return "parameter clause"
     case .returnClause:
-      return "return clause"
```

Comment:
What diagnostic did you not want to have `return clause`? IMO diagnostics with "after return" do make sense.

Reply:
```swift
struct Foo {
  subscript(x: String) {}
}
```

If we keep `return clause` as a node name we emit
```
Expected return type in return clause
```

instead of
```
Expected return type in subscript
```

I liked the second better than the first.

Reply:
Oh sorry, it's the return clause within a declaration. Sure, definitely agree with you there 👍